src/sagemaker/session.py

```diff
@@ -132,6 +132,7 @@ def upload_data(self, path, bucket=None, key_prefix='data'):
                 creates it).
             key_prefix (str): Optional S3 object key name prefix (default: 'data'). S3 uses the prefix to
                 create a directory structure for the bucket content that it display in the S3 console.
+            ExtraArgs (dict): Extra arguments that may be passed to the upload operation.
```

Comment:
let's link to the AWS docs here so that users know where to find what extra arguments are allowed

Reply:
Linked to ExtraArgs documentation